setup.py

```diff
@@ -35,5 +35,5 @@ def package_data(pkg, root_list):
     entry_points={
         'xblock.v1': 'drag-and-drop-v2 = drag_and_drop_v2:DragAndDropBlock',
     },
-    package_data=package_data("drag_and_drop_v2", ["static", "templates", "public"]),
+    package_data=package_data("drag_and_drop_v2", ["static", "templates", "public", "translations"]),
```

Comment:
@mattdrayer is it required to update the version in setup.py for this xblock  ? currently it is pointing to `2.0.4`